debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java

```diff
@@ -605,12 +602,11 @@ public static Builder create() {
     private SourceTask task;
     private final Transformations transformations;
 
-    private EmbeddedEngine(Configuration config, ClassLoader classLoader, Clock clock, DebeziumEngine.ChangeConsumer<SourceRecord> handler,
+    private EmbeddedEngine(Configuration config, Clock clock, DebeziumEngine.ChangeConsumer<SourceRecord> handler,
```

Comment:
Not sure what is the best way to remove `ClassLoader classLoader,` from argument.. its not necessary anymore

Reply:
@ismailsimsek I tend to disagree in this one. Tha classloaader should still be allowed to be set externally. Just if not set, it will be obtained from Instantiator class.

Reply:
ok, updated